### PR TITLE
remove GPDB_12_MERGE_FIXME in tidbitmap.h: make sure TBM_BITS_PER_BITMAPWORD is 64-bit

### PR DESCRIPTION
**Pull request by SmartKeyerror**
In PostgreSQL 12, it will use the size of a pointer to define the size of a bitmap word, just 
like:

```
/*
 * Data representation
 *
 * Larger bitmap word sizes generally give better performance, so long as
 * they're not wider than the processor can handle efficiently.  We use
 * 64-bit words if pointers are that large, else 32-bit words.
 */
#if SIZEOF_VOID_P >= 8

#define BITS_PER_BITMAPWORD 64
typedef uint64 bitmapword;		/* must be an unsigned type */
typedef int64 signedbitmapword; /* must be the matching signed type */

#else

#define BITS_PER_BITMAPWORD 32
typedef uint32 bitmapword;		/* must be an unsigned type */
typedef int32 signedbitmapword; /* must be the matching signed type */

#endif
```

So, `bitmapword` will be 32 bits on some 32-bit-platform.

For the functions and methods in the `bitmap.c` file, the length of the bitmap only affects its 
efficiency, not the correctness of its functions. But for the bitmap in the bitmap index, using 
32 bits can lead to fundamental errors.

This is because in the implementation of the greenplum bitmap index, the bitmap compresses 
it using the HRL method as follows:

Before compression:
````
before compression:
00000000 00000000 01000000 11111111 11111111 11111111

after compression using HRL method:
Header Section: 101
Content Section: 00000010 01000000 10000011
````

We judge whether a word in the Content Section is compressed by judging the bit in the Header 
Section, 1 is compressed, 0 is not compressed. At the same time, the first bit of the compressed 
word indicates whether the value before compression is 0 or 1, and **the rest represents the original 
length of the value before compression**.

So, if we use a 32-bit bitmap, then a compressed word can represent a maximum length of 
`2^(32 - 1) = 2147483648` , so there is a risk of overflow, resulting in data errors.

In summary, we must use a 64-bit bitmap to avoid the compressed word length overflow problem.
We cannot use `BITS_PER_BITMAPWORD` in `tidbitmap.h`

### CODE REVIEW

**Review by interma — APPROVED** [review]
LGTM

**Comment by interma** [comment]
Agreed with you, and the default HRL word length of bitmap index is also 64bit.
https://github.com/greenplum-db/gpdb/blob/61323b178e46b8a9595f06e14cff9c91bd2f144a/src/include/access/bitmap.h#L24-L32

Changing it may need to denote big effort to deal corner case.

**Comment by SmartKeyerror** [comment]
This PR just change the comment, and confirmed that's impossible to use 32-bit word in greenplum bitmap index, so merge it.
